next.config.js

```diff
@@ -9,27 +9,32 @@ module.exports = {
         hostname: process.env.NEXT_IMAGE_DOMAIN,
       },
     ],
+    deviceSizes: [
+      // 640,
+      750,
+      // 828,
+      // 1080,
+      1200,
+      // 1920,
+      // 2048,
+      3840
+    ],
+    imageSizes: [
+      // 16,
+      // 32,
+      // 48,
+      // 64,
+      // 96,
+      128,
+      // 256,
+      384
+    ],
```

Comment:
What unit are these in? I'm not fully grokking the documentation 

https://nextjs.org/docs/app/api-reference/components/image#imagesizes

https://nextjs.org/docs/app/api-reference/components/image#sizes

Do you have an example I can follow?

Reply:
https://html.com/attributes/img-srcset/#Specifying_Image_Width

So it tells the browser the size of the image... which isn't really accurate, but it's something
```
<img alt="" loading="lazy" decoding="async" data-nimg="fill" class="su-object-cover su-object-center" style="position: absolute; height: 100%; width: 100%; inset: 0px; color: transparent;" sizes="100vw" srcset="https://example.com/breakpoint_lg_2x/image.jpg 750w, https://example.com/breakpoint_xl_2x/image.jpg 1200w, https://example.com/breakpoint_2xl_2x/image.jpg 3840w" src="https://example.com/breakpoint_2xl_2x/image.jpg">
```